src/main/java/frc/robot/subsystems/drivetrain/commands/SwitchGears.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.drivetrain.Drivetrain;
+
+public class SwitchGears extends InstantCommand {
+    private Drivetrain drivetrain;
+    private boolean initialPiston;
+
+    public SwitchGears(Drivetrain drivetrain) {
+        this.drivetrain = drivetrain;
+        addRequirements(drivetrain);
+        initialPiston = drivetrain.getPiston();
+    }
+
+    @Override
+    public void initialize() {
+        drivetrain.togglePiston();
```

Comment:
Add check that the robot is moving, the gear shifter can be damaged if the robot is static.